examples/shared_samples/storage_system.rb

```diff
@@ -21,6 +21,7 @@
 # - API300 for C7000 and Synergy
 # - API500 for C7000 and Synergy (see /examples/api500/storage_system.rb)
 # - API600 for C7000 and Synergy (see /examples/api600/storage_system.rb)
+# - API1600 for C7000 and Synergy (see /examples/api600/storage_system.rb)
```

Comment:
please mention for other API versions also

Reply:
done